data/governments.txt

```diff
@@ -650,6 +651,78 @@ government "Heliarch"
 
 color "governments: Independent" .78 .36 .36
 
+# Korath-friendly, pirate-friendly, human, settlements or stations that dislike the Syndicate
+government "Humanika (Port)"
```

Comment:
There's already a Quarg planet named Humanika with a substantial human population. I would probably change the name of this government to something slightly different in order to avoid confusion.

Reply:
I was wondering when someone would make that complaint. "Humanika" literally means "human."

Coming from a Korath, "Humanika Rafeki" has the literal translation of "human friends." To the Korath, we are "Humanika," and the status of some humans as friends will occur several times. You will see "Humanika" frequently throughout the campaign.

The Quarg may be so brazenly uncreative as to name a planet "human," but that won't stop the Korath from using it for it's intended purpose: describing humans.

Reply:
In korath missions.txt the Korath greet you as "Humani efreti". I believe 'efreti' was established to be the Korath word for 'friend', and judging from that passage the Korath word for human is 'humani' instead of 'humanika'. I might be wrong though and there's something about the Korath language I'm missing.

Reply:
It is also possible that my knowledge of the Kor Efret is not as good as it should be. However, the Korath Exile word for "human" is "Humanika" according to @ravenshining

Reply:
It's also in the `First Contact: Korath Exiles` mission, which dates back pretty far. In all likelihood, this name came from MZ. Recall that the Korath and Quarg were close friends for thousands of years, and most likely knew about humans from Quarg stories before meeting one in real life.

```
Akafavah says, "Oy! Humanika!" It's not clear what emotion they're feeling, but it's a strong one.
```

Reply:
I still think having a government and a planet named Humanika that have nothing to do with each other is needlessly confusing. Slightly changing the Exile name for human in order to avoid this is worth it in my opinion.

Reply:
https://github.com/endless-sky/endless-sky/commit/69e391572b416ed56d1a5865253621fc7351337c

This is what I've found of the old mission, doesn't look like that comes up there? Unless it was developed further and I'm missing that.

But yeah, feels like it'd be the opposite, where the Efret being close with the Quarg would adopt some of their terms, while the Exiles would develop their own. Unless what would become the Exiles were in much closer/more-extended contact with the Quarg than I thought.

Reply:
They aren't unrelated, and, I like the obvious implication that the two names have the same origin, as it makes the player suspect a connection between the Korath and the Quarg. It would be a good lead-in to the Korath giving their back-story of interactions with the Quarg. The two species were quite close friends before the Quarg abandoned the Korath during their hour of need.

Reply:
If it's important for lore reasons that the Quarg and Korath names for human are the same, then we might want to rename Humanika the planet to something a little more in depth than just 'human'. Not sure if that would cause save file issues though.

Reply:
> Unless what would become the Exiles were in much closer/more-extended contact with the Quarg than I thought.

Keep in mind, these are the Korath who preferred to risk starvation so they could keep their religions. Religions that have passing of knowledge through skin and feather as a core tenet. They would have an excellent memory of the distant past, and the role the Quarg played in the Korath being sent into starving exile. When they see a human, and remember the Quarg speaking of them hundreds of years ago, I'd expect the word would immediately click into their minds rather than inventing a new one.

As for the Kor Efrit, I haven't given them much thought since they aren't a significant part of the campaign plans, so far as I know.

Reply:
I do want the names to be the same, so the player will suspect a close connection between the two races, and seek to investigate it.

If we need a detailed lore explanation, I see this one: the Korath met a hairless biped, and remembered descriptions from their old "friends," the Quarg. They looked up the details in their databases of that time, and saw "Humanika" and went with it.

Reply:
I'm going to start on the translator story next, and I think a good addition to that is to get some information from the Korath, including the story about "Humanika" and the Quarg. It could be triggered by visiting that system, so you have the planet on your map.

Reply:
> If it's important for lore reasons that the Quarg and Korath names for human are the same, then we might want to rename Humanika the planet to something a little more in depth than just 'human'. Not sure if that would cause save file issues though.

Given the paucity of content that involves Humanika, I suspect there shouldn't be any major problems tweaking it unless someone happens to be on the planet when they update their game.


As far as the names go, I think having a name that starts out confusing but later leads to an "Ah ha!" sort of realization is a neat thing to have, as is allowing the player to discover how other cultures have cross-contaminated ideas out there without needing First Last to be the bridge/contact point.

Reply:
> As far as the names go, I think having a name that starts out confusing but later leads to an "Ah ha!" sort of realization is a neat thing to have, as is allowing the player to discover how other cultures have cross-contaminated ideas out there without needing First Last to be the bridge/contact point.

Perhaps we could add another word to Humanika? Something like Humanika (insert Quarg word for planet here)? That way there's a clear distinction between the two but the link between the Quarg and Korath words for human is still maintained.

Reply:
How about the quarg word for "Planet" followed by "Humanika." Then it becomes "Planet of Humans."

Reply:
I'm going to leave the Quarg planet name as it is for now, while the Korath translator mission sequence takes shape. A tentative plan for that is for the player to find things around the galaxy that appear connected to the Korath, and then go back to the Korath to ask for their side of the story. The first time you do this, they send you to get a translator. Once you have one, you can get their side of the story, of whatever it is you find.

Korath pride themselves in their individuality, self-expression, and artistic talent. I think having them look down on Quarg as being a largely uncreative species would add an interesting dynamic to the Quarg-Korath relationship. When Captain Last describes to them a Quarg planet that is so uncreatively called "Humans," they'll reveal their disdain.